coordinator/src/main/scala/com/socrata/datacoordinator/service/collocation/Collocator.scala

```diff
@@ -250,6 +250,12 @@ class CoordinatedCollocator(collocationGroup: Set[String],
                   }.headOption.getOrElse(Approved)
                 }
 
+              totalStatus match {
+                case Rejected(reason) => log.info("Rejecting collocation request because {}", reason)
+                case Approved => log.info("Approving collocation request")
```

Comment:
Since you're adding logging for this could you also get some info about what is getting collocated